wordpress-mobile/WordPress-iOS
# Remove injected CSS/JS that is no longer needed

**Fixes** https://github.com/wordpress-mobile/WordPress-iOS/issues/15411

### To test:
1. Navigate to the Create WordPress.com site
    - My Sites > ➕ > Create WordPress.com site
1. Choose a Design or Select Skip
1. Search for and choose a domain
1. Select Create Site
1. Wait for your site to be created and for the preview to load
1. **Expect** 
    - To not see a header like the admin bar, the cookie notice or the create a page on WordPress.com notice
    - You also shouldn't be able to click any of the buttons or links on the preview webview. 

### PR submission checklist:
